src/Tools/prec_anaone.m

```diff
@@ -25,60 +25,56 @@
 % B         : relative lag amplitude according to Eq. 2
 % ac       	: cumulative autocorrelation 
 
+% DEPENDENT FUNCTIONS:
+% prec_de_conv.m    = deconvolution
+% prec_rev_conv.m   = convolution with reversed IR
+% prec_peakratio.m  = determines relative lead/lag ratio from autorcorrelation function
+% prec_reconHW.m    = reconstruction of full wave from halfwave rectified signal
+% calcXCorr.m       = Cross-correlation calculation
+
 Fms=Fs./1000; % Sampling frequency based on milliseconds
-% lags=round(10.*Fms); % number of lags for autocorrelation process 
 lags=maxLag; % number of lags for autocorrelation process 
-MainPeakWidth=0.75.*Fms; % max main peak width to determine minimum lag
-
-    
+MainPeakWidth=floor(0.75.*Fms); % max main peak width to determine minimum lag
 
```

Comment:
In the mean time in the master `round(0.75.*Fms)` was added, which causes the remaining conflict of this pull request.
@ryanckim: could you decide which one we should use `round()` or `floor()`.